src/app/location/location.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <h1>Location Test</h1>
+  <form #cityForm="ngForm" (ngSubmit)="onSubmit(cityForm)">
```

Comment:
not sure about this. We should create a form on the init of the component probably. I'm not familiar with the `#someForm='ngForm'`. @BartlomiejWojcik do you know anything about this?

Reply:
whole this file probably would be rewritten for our designs